.changeset/chatty-months-report.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Similarly, if you rename this `techdocs-chatty-months-report.md`, we can avoid maintainer review.